package/MDAnalysis/converters/OpenMMParser.py

```diff
@@ -73,6 +76,11 @@
     Segids,
 )
 
+try:
+    from openmm.unit import daltons
+except ImportError:
+    from simtk.unit import daltons
+
```

Comment:
@Atharva7K the tests are failing here because in minimal environments, OpenMM is not installed. Where do you use daltons? It would be better to just import it there.

Reply:
Umm I'm using daltons in the OpenMMParser to convert the mass using `elem.mass.value_in_unit(unit.daltons)` as you requested but I'm not sure how that can be done without importing daltons in the parser file. But since amu and dalton is [equal](https://github.com/openmm/openmm/blob/be19e0222ddf66f612016a3c1f687161a53c2396/wrappers/python/openmm/unit/unit_definitions.py#L213) in OpenMM, will it be fine to use the private `mass._value` like before?

Reply:
Yes, you'll still need to import `daltons`, but doing it at the top level without a safeguard is leading to the error. There are two options here:

* add an `except ImportError` to the `simtk.units` that `pass`es if OpenMM is not installed, e.g. https://github.com/MDAnalysis/mdanalysis/blob/459c04ea86fc9f885d0c259e831a735111c249ce/package/MDAnalysis/converters/RDKit.py#L97-L101
* import `daltons` in the function where you use it

Currently it's erroring because the converter class is imported into the main package even when OpenMM is not installed, which runs into this top-level import. There's a similar function-level import in the RDKit package: https://github.com/MDAnalysis/mdanalysis/blob/459c04ea86fc9f885d0c259e831a735111c249ce/package/MDAnalysis/converters/RDKit.py#L322-L338